benchmarks/multiplatform/benchmarks/src/commonMain/kotlin/benchmarks/lazygrid/LazyGrid.kt

```diff
@@ -67,30 +50,31 @@ fun LazyGrid() {
                 curItem = state.firstVisibleItemIndex
                 if (curItem == 0) direct = true
                 if (curItem > itemCount - 100) direct = false
-                state.scrollBy(if (direct) 5f else -5f)
+                state.scrollBy(if (direct) 55f else -55f)
             }
         }
     } else {
-        LaunchedEffect(curItem) {
-            withFrameMillis { }
-            curItem += if (direct) 50 else -50
-            if (curItem >= itemCount) {
-                direct = false
-                curItem = itemCount - 1
-            } else if (curItem <= 0) {
-                direct = true
-                curItem = 0
+        LaunchedEffect(Unit) {
+            while(isActive) {
```

Comment:
as smoothScroll is not changing by checkbox now, we can use isActive for the similar loop above, I guess.

Reply:
fixed